### ISSUE TITLE
Add support for GDAL 2.0

### ISSUE THREAD

**Issue by joto** [osmcode/osmcoastline#15]

**Comment by sebastic** [comment]
As reported in [Debian Bug #802813](https://bugs.debian.org/802813), osmcoastline fails to build with GDAL 2.0:
```
 error: invalid conversion from 'GDALDriver*' to 'OGRSFDriver*'
```
The full [build log](https://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;att=1;bug=802813;filename=osmcoastline_2.1.1-1_gdal-2.0.log) is available there too.

osmcoastline needs to be patched to support GDAL 2.0, please refer to the
[MIGRATION GUIDE FROM GDAL 1.11 to GDAL 2.0](https://svn.osgeo.org/gdal/branches/2.0/gdal/MIGRATION_GUIDE.TXT) for pointers.

**Comment by sebastic** [comment]
Since #16 isn't going to be merged, can you give an indication when you expect a new release that includes GDAL 2.0 support using [gdalcpp](https://github.com/joto/gdalcpp)?
